es6-promises/main.js

```diff
@@ -0,0 +1,11 @@
+import takeAChance from './take-a-chance.js';
+
+takeAChance('Tim')
+  .then((value) => {
+    console.log(value);
+  }
```

Comment:
Usually to format this we would put the closing `)` on the last line of the function:
```
takeAChance('Tim')
  .then((value) => {
    console.log(value);
  })
  .catch((error) => {
      console.error(error.message);
  });
```